composer.json

```diff
@@ -40,8 +40,8 @@
     },
     "require-dev": {
         "json-schema/JSON-Schema-Test-Suite": "1.1.0",
-        "phpunit/phpunit": "~3.7",
-        "phpdocumentor/phpdocumentor": "~2"
+        "phpdocumentor/phpdocumentor": "~2",
+        "phpunit/phpunit": "^4.7"
```

Comment:
@GrahamCampbell 

Requiring `phpunit/phpunit` without any version constraint yields to what you can see here now!